src/html/blocks/week-calendar-day.html

```diff
@@ -0,0 +1,5 @@
+<input class="week-calendar__radiobutton" type="radio" name="@@name" id="@@input" @@ischecked />
+<label class="week-calendar-day" for="@@input">
```

Comment:
I think, will be better use name "week__item" or "week__day" instead "week-calendar-day".